### DISCUSSION TITLE
Expanding the Project

### DISCUSSION THREAD

**Discussion by sulan-blob** [jbevain/cecil#746] [Q&A] [answered]
Hi,

I would like to expand the Project, because I want to write a Tracer. But before I do, I would first verify something. Is it currently possible to get the values out of method paremters? Because I would like to re-use them and build a AOP kind Tracer, by logging/printing them before and afterwards of a method body. Is it something which can be achieved?

T&R,
Sulan

**Comment by jbevain** [comment, marked as answer]
Hi,

Yes it's possible to use Cecil to build a “AOP kind tracer”.

It seems that you're quite early in your project, you can look at the tests or at the Fody project to learn more about using Cecil.
